README.md

```diff
@@ -45,28 +45,34 @@ python import_video.py --help
 Which will output:
 
 ```man
-usage: import_video.py [-h] [-r REGEX] [-c] [-s STORAGE] [-f FOLDER] [-i HOST]
-                   [-p PORT] [-m HOOK_MODULE] [-d HOOK_DATA_JSON] [-v] [-q]
+usage: import_video.py [-h] [-v] [-q] [-c] [-p PORT] [-r REGEX] [-s STORAGE]
+                       [-d HOOK_DATA_JSON] [-f HOOK_DATA_JSON_FILE]
+                       folder hook_module host
+
+positional arguments:
+  folder                full path to folder of input videos to process
+  hook_module           full path to hook module for custom functions (a
+                        python file)
+  host                  the IP-address / hostname of the segmenter
 
 optional arguments:
   -h, --help            show this help message and exit
-  -r REGEX, --regex REGEX
-                        regex to find camera name
+  -v, --verbose         set logging level to debug
+  -q, --quiet           set logging level to errors only
   -c, --csv             dump csv log file
+  -p PORT, --port PORT  the segmenter port number (default: 8080)
+  -r REGEX, --regex REGEX
+                        regex to find camera name (default:
+                        .*/(?P<camera>.+?)/(?P<epoch>\d+(.\d+)?).*)
   -s STORAGE, --storage STORAGE
-                        location of the local storage db
-  -f FOLDER, --folder FOLDER
-                        folder to process
-  -i HOST, --host HOST  the segmenter ip
-  -p PORT, --port PORT  the segmenter port number
-  -m HOOK_MODULE, --hook_module HOOK_MODULE
-                        full path to hook module for custom functions (a
-                        python file)
+                        full path to the local storage db (default:
+                        ./processes.shelve)
   -d HOOK_DATA_JSON, --hook_data_json HOOK_DATA_JSON
                         a json object containing extra information to be
                         passed to the hook-module
-  -v, --verbose         set logging level to debug
-  -q, --quiet           set logging level to errors only
+  -f HOOK_DATA_JSON_FILE, --hook_data_json_file HOOK_DATA_JSON_FILE
+                        full path to a file containing a json object of extra
+                        info to be passed to the hook module
 ```
 
 The video-importer will traverse a directory to extract the camera-name and video-timestamp from the video filenames,
```

Comment:
@jhallard clarify "camera-name" and "video-timestamp", since those aren't variable names used elsewhere. Maybe say:
The video-importer will traverse a directory to extract the camera name and video timestamp for each video filename.